tests/output/LoggingTest.py

```diff
@@ -3,6 +3,16 @@
 import unittest
 
 from coalib.output.Logging import configure_json_logging
+from coalib.output.printers.LogPrinter import LogPrinterMixin
+
+
+class TestLogger(LogPrinterMixin):
+
+    def __init__(self):
+        pass
+
+    def log_message(self, log_message, **kwargs):
+        super(TestLogger, self).log_message(log_message)
```

Comment:
separate commit or does this test belong to the current commit? (is this maybe for coverage?)